src/developer/web-api/tracker.md

```diff
@@ -506,6 +506,7 @@ Table: Tracked entity instances query parameters
 | assignedUser | Filter the result down to a limited set of teis with events that are assigned to the given user IDs by using *assignedUser=id1;id2*.This parameter will be considered only if assignedUserMode is either PROVIDED or null. The API will error out, if for example, assignedUserMode=CURRENT and assignedUser=someId |
 | trackedEntityInstance | Filter the result down to a limited set of teis using explicit uids of the tracked entity instances by using *trackedEntityInstance=id1;id2*. This parameter will at the very least create the outer boundary of the results, forming the list of all teis using the uids provided. If other parameters/filters from this table are used, they will further limit the results from the explicit outer boundary. |
 | includeDeleted | Indicates whether to include soft deleted teis or not. It is false by default. |
+| potentialDuplicate | Indicates if a tei is a Potential Duplicate. Can be true &#124; false or omitted. If omitted, we don't check whether a tei is a potentialDuplicate or not. |
```

Comment:
I think it is important to highlight that this is a filter.
Maybe something like `Filter the result based on the fact tei is a Potential Duplicate`